service/controller/resource/scrapeconfigs/test/aws/case-0-cluster-api.golden

```diff
@@ -35,6 +38,9 @@
   # Add provider label.
   - target_label: provider
     replacement: aws
+  # Add installation label.
+  - target_label: installation
+    replacement: 
```

Comment:
Should this be empty?

Reply:
No, the cm is probably empty

Reply:
@TheoBrigitte I am not sure how I should set this one up. Should it be generated? Do you have a clue?